volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -539,3 +545,248 @@ def reference_count(self):
             ret = refcnt.refs.counter
 
         return ret
+
+class mnt_namespace(objects.StructType):
+    def get_inode(self):
+        if self.has_member("proc_inum"):
+            return self.proc_inum
+        elif self.ns.has_member("inum"):
+            return self.ns.inum
+        else:
+            raise AttributeError("Unable to find mnt_namespace inode")
+
+class net(objects.StructType):
+    def get_inode(self):
+        if self.has_member("proc_inum"):
+            return self.proc_inum
+        elif self.ns.has_member("inum"):
+            return self.ns.inum
+        else:
+            raise AttributeError("Unable to find net_namespace inode")
+
+class sock(objects.StructType):
+    def __get_vol_kernel_module_name(self):
```

Comment:
why does this start with two underscores @gcmoreira?

Reply:
fixed

Reply:
Actually, I realized when I wrote the comment [below](https://github.com/volatilityfoundation/volatility3/pull/597#discussion_r815225600) that it would probably better to move this function to somewhere it can be called by both `LinuxUtilities::_get_new_sock_pipe_path` in [these lines](https://github.com/volatilityfoundation/volatility3/blob/9a0ba988abe8028a2a638bce4d9a8e43a11d7c4e/volatility3/framework/symbols/linux/__init__.py#L115-LL124) and by `sock::inode` [here](https://github.com/volatilityfoundation/volatility3/blob/a7df7617dd68edd7e3bd3696b12a8bd6d3df3c51/volatility3/framework/symbols/linux/extensions/__init__.py#L595) . Thoughts?

Reply:
I do think one place for it is best, but I am not sure where that would be. What you do think @ikelos ?

Reply:
in particular, all the extensions inherit from objects.StructType, so there is not a super class we can place it without changing them all.

Reply:
We can slide one in between (so `class LinuxSocket(objects.StructType)`), in which case it would probably live under `symbols/linux/extensions` I guess?  It we just want generic code accessible by anyone, then probably the same place, but in a file that needs explicitly importing.